docker-compose.selenium.yml

```diff
@@ -1,14 +0,0 @@
-version: "2.1"
```

Comment:
Hmm, I found it very useful sometimes for debugging. The visual output helped with nasty feature tests.

Reply:
But still possible to create screenshots in the tests, so I guess I could live without it